api/src/main/java/org/cardanofoundation/rosetta/common/util/ParseConstructionUtil.java

```diff
@@ -295,6 +302,18 @@ public static List<Operation> parseCertsToOperations(TransactionBody transaction
     return parsedOperations;
   }
 
+  private static String getStakingCredentialHex(Operation certOperation) {
+    String hex = null;
+    if (checkStakeCredential(certOperation)) {
+      hex = certOperation.getMetadata().getStakingCredential().getHexBytes();
```

Comment:
Can be null here?

Reply:
No, it can't be. In the method `checkStakeCredential` we checked that.